app/styles/layout/_header.scss

```
@@ -0,0 +1,205 @@
+.header {
+  height: 3.5rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  background: rgba(250, 250, 250, 0.1);
+
+  &-container {
+    height: 100%;
+    @include flex(center, space-between);
+
+    @include responsive(LG) {
+      @include flex(center, flex-end);
```

```suggestion
      @include flex($justify: flex-end);
```
Nếu chỉ muốn thay đổi 1 mình justify-content thôi thì chỉ cần như này nhé